diff --git a/index.d.ts b/index.d.ts
index aad07ffdb108913eb8b40d47ef0db704442bf847..11e13d38dd52ccb1a4a24665be50915a31b8d870 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -66,6 +66,15 @@ declare namespace GulpClient {
          * @default true
          */
         queue?: boolean | undefined;
+        /**
+         * An event name or array of event names to listen for. Useful if you only need to watch specific events.
+         * @src
+         *   gulp.watch is imported from glob-watcher (see https://github.com/gulpjs/gulp/blob/v4.0.2/index.js lines 6, 28 and 48)
+         *   gulp use glob-watcher@^5.0.3 (see https://github.com/gulpjs/gulp/blob/v4.0.2/package.json#L33)
+         *   glob-watcher declare publicly options.events (see https://github.com/gulpjs/glob-watcher/blob/v5.0.3/README.md?plain=1#L101)
+         * @default ['add','change','unlink']
+         */
+        events?: string | string[];
     }
 
     interface WatchMethod {
